### DISCUSSION TITLE
Windows 10 LTSC Support

### DISCUSSION THREAD

**Discussion by methanoid** [MustardChef/WSABuilds#86] [Q&A] [answered]
Will any of these builds work on LTSC? Its SOOOOO much nicer than stock W10

**Comment by MustardChef** [comment, marked as answer]
> Will any of these builds work on LTSC? Its SOOOOO much nicer than stock W10

What is your Windows 10 build number for LTSC.
You can give them a try, although I am not 100% sure that these will work on LTSC builds of Windows 10. If you are going to give WSA a shot, make sure that you have the latest updates installed!

**Comment by methanoid** [comment]
21H2 build 19044  (didnt realise it was THAT new!)

Given the millions of options here, if you can navigate me to one WITH GApps and WITHOUT Amazon and possible WITH Magisk (not sure I need it but used on Android phones many times) I'd be be sorted :-)

**Reply by MustardChef** [reply]
@methanoid Well, the first build in the list of assets, has Google Play Services and No Amazon AppStore (as seen below) and the last build has Google Play Services and No Amazon AppStore and Magisk (as seen below).

|****Operating System****|****Download Page****|****Download Mirrors****|
|----------|-----------|--------------|
|<img src="https://upload.wikimedia.org/wikipedia/commons/0/05/Windows_10_Logo.svg" style="width: 200px;"/> | [![win10down](https://img.shields.io/badge/Download%20Latest%20Build-Windows%2010%20x64-blue?style=for-the-badge&logo=windows)](https://github.com/MustardChef/WSABuilds/releases/tag/Windows_10_2303.40000.4.0)|[<img src="https://img.shields.io/badge/OneDrive-white?style=for-the-badge&logo=Microsoft%20OneDrive&logoColor=0078D4" style="width: 150px;"/>](https://x6cgr-my.sharepoint.com/:f:/g/personal/mcdt_x6cgr_onmicrosoft_com/Elgd2h-5NLtAm33Ox_KsJEQBdKAur5k5OVBgJ_HUd79F3g?e=9bHfd2)|

![image](https://github.com/MustardChef/WSABuilds/assets/68516357/92fc084a-9d67-4132-a9da-7d8d210707c6)

**Reply by MustardChef** [reply]
@methanoid Please can you feedback to me if WSA worked on Windows 10 LTSC or not.

**Reply by sudozid** [reply]
it shows this for me 

`
Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF9, Install failed. Please contact your software vendor.
(Exception from HRESULT: 0x80073CF9)
error 0xCE: Creating the package metadata directory C:\WSA_2303.40000.5.0_x64_Release-Nightly-with-magisk-d0c93842.2610
1.-canary-MindTheGapps-13.0-RemovedAmazon_Windows_10\WSA_2303.40000.5.0_x64_Release-Nightly-with-magisk-d0c93842(26101)
-canary-MindTheGapps-13.0-RemovedAmazon\microsoft.system.package.metadata failed.
NOTE: For additional information, look for [ActivityId] d196c3ee-97c4-000f-97d3-96d1c497d901 in the Event Log or use
the command line Get-AppPackageLog -ActivityID d196c3ee-97c4-000f-97d3-96d1c497d901
At line:1 char:1
+ Add-AppxPackage -ForceApplicationShutdown -ForceUpdateFromAnyVersion  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\WSA_2303.400...ppxManifest.xml:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand`

**Reply by MustardChef** [reply]
> it shows this for me
> 
> ` Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF9, Install failed. Please contact your software vendor. (Exception from HRESULT: 0x80073CF9) error 0xCE: Creating the package metadata directory C:\WSA_2303.40000.5.0_x64_Release-Nightly-with-magisk-d0c93842.2610 1.-canary-MindTheGapps-13.0-RemovedAmazon_Windows_10\WSA_2303.40000.5.0_x64_Release-Nightly-with-magisk-d0c93842(26101) -canary-MindTheGapps-13.0-RemovedAmazon\microsoft.system.package.metadata failed. NOTE: For additional information, look for [ActivityId] d196c3ee-97c4-000f-97d3-96d1c497d901 in the Event Log or use the command line Get-AppPackageLog -ActivityID d196c3ee-97c4-000f-97d3-96d1c497d901 At line:1 char:1
> 
>     * Add-AppxPackage -ForceApplicationShutdown -ForceUpdateFromAnyVersion  ...
> 
>     * ```
>         + CategoryInfo          : WriteError: (C:\WSA_2303.400...ppxManifest.xml:String) [Add-AppxPackage], IOException
>         + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand`
>       ```

@sudozid Please follow this fix guide: https://github.com/MustardChef/WSABuilds/blob/master/Fixes/Fix%20Error%200x80073CF9.md

**Reply by sudozid** [reply]
thank you that was simple, it works on LTSC!

**Comment by angelherver** [comment]
cool

**Comment by ykingraj** [comment]
can i install apks in wsa ?

**Reply by MustardChef** [reply]
@ykingraj Yes, please check this guide: https://github.com/MustardChef/WSABuilds/blob/master/Guides/Sideloading.md.
